<ui version="4.0" >
 <class>MyColorPickerClass</class>
 <widget class="QDialog" name="MyColorPickerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MyColorPicker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QtColorTriangle" native="1" name="colortriangle" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="7" >
      <widget class="QLineEdit" name="redEdit" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="7" >
      <widget class="QLineEdit" name="greenEdit" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="7" >
      <widget class="QLineEdit" name="blueEdit" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="6" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Red:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="6" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Green:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="6" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Blue:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="0" column="8" >
      <widget class="QLabel" name="viewCol" >
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="htmlColEdit" />
     </item>
     <item row="2" column="5" >
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item rowspan="2" row="2" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="oldBut" >
         <property name="text" >
          <string>Old Picker</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <widget class="QCheckBox" name="proofCheck" >
       <property name="text" >
        <string>Proof</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="7" >
      <widget class="QLineEdit" name="alphaEdit" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="6" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Alpha:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QPushButton" name="invalidBot" >
       <property name="text" >
        <string>Set Invalid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QtColorTriangle</class>
   <extends>QWidget</extends>
   <header>src/MyLib/QtAdd/qtcolortriangle.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
